ui/js/directives/treeherder/clonejobs.js

```diff
@@ -526,29 +549,33 @@ treeherder.directive('thCloneJobs', [
 
             //Empty the job column before populating it
             jobTdEl.empty();
-
-            var jgObj, jobGroup, i;
-            for (i=0; i<jobGroups.length; i++) {
-
-                jgObj = jobGroups[i];
-
+            var jgObj, jobTdHtml = "";
```

Comment:
If you look carefully, you'll see that the jgObj temporary is no longer necessary, and we can just replace all instances of it with `jobGroup`.